src/Resources/contao/controllers/BackendMain.php

```diff
@@ -268,6 +252,8 @@ protected function output()
 		$this->Template->systemMessagesCount = substr_count($strSystemMessages, 'class="tl_');
 		$this->Template->systemErrorMessagesCount = substr_count($strSystemMessages, 'class="tl_error"');
 
+		$this->setImporsonatedLogout();
```

Comment:
Typo: imp**e**rsonated

Reply:
Fixed in 2741c66253622f16f2eea2f85ab0a78f48d2ffd7